lib/common/graph.js

```diff
@@ -170,28 +169,26 @@ class DirectedGraph {
  * @class
  * @memberof module:concerto-util
  */
-class ConcertoGraphVisitor extends DiagramVisitor {
+class ConcertoGraphVisitor extends BaseVisitor {
     /**
      * Visitor design pattern
      * @param {ClassDeclaration} classDeclaration - the object being visited
      * @param {Object} parameters  - the parameter
      * @protected
      */
     visitClassDeclaration(classDeclaration, parameters) {
-        parameters.stack ??= [];
-        parameters.stack.push(classDeclaration.getFullyQualifiedName());
         parameters.graph.addVertex(classDeclaration.getFullyQualifiedName());
 
         if (classDeclaration.getSuperType()){
             parameters.graph.addEdge(classDeclaration.getFullyQualifiedName(), classDeclaration.getSuperType());
-            // this "if" block adds the types that extend the Super type
+
+            // Adds a reverse edge to create a cyclic dependency between a class declaration and its supertype.
             if(!parameters.createDependencyGraph && classDeclaration.getSuperType() !== 'concerto@1.0.0.Concept') {
```

Comment:
Is this parameter documented anywhere?

Reply:
>  I think we should rename the parameter to includeDerivedTypes

This is a breaking change for anyone relying on `createDependencyGraph`.

Reply:
If it is undocumented there can't be many uses. Can you try to track them down? I guess we could keep the negative aspect and call it `excludeDerivedTypes`  so the default behaviour (no options set) doesn't change?

Reply:
After reviewing this, I think if we are renaming the property, then a term that conveys a truthy state would be more effective. I’d suggest keeping `includeDerivedTypes`.